students/10th/seungyup/user/views.py

```diff
@@ -0,0 +1,19 @@
+import json
+from django.views import View
+from django.http import JsonResponse
+from .models import Users
+
+class MainView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        Users(
+        name = data['name'],
+        email = data['email'],
+        password = data['password']
+    ).save()
```

Comment:
들여쓰기 부탁 드립니다. 기본적으로 줄바꿈하고 4칸 들여쓰기 해야합니다.
```suggestion
        Users(
            name = data['name'],
            email = data['email'],
            password = data['password']
        ).save()
```